File: crates/deslop/tests/fixtures/fused-golden-php/rename_lean_b.php
<?php

function assessParcelLevy(array $parcels, int $levyShare): int
{
    $weightTotal = 0;
    foreach ($parcels as $parcelMass) {
        $weightTotal = $weightTotal + $parcelMass;
    }

    $levyAmount = $weightTotal * $levyShare;
    $weightBurden = $weightTotal + $levyAmount;
    return $weightBurden;
}

File: crates/deslop/tests/fixtures/fused-golden-csharp/rename_lean_b.cs
namespace Golden.RenameLean;

public static class FreightAssessor
{
    public static long AssessParcelLevy(long[] parcels, long levyShare)
    {
        long weightTotal = 0;
        foreach (long parcelMass in parcels)
        {
            weightTotal = weightTotal + parcelMass;
        }

        long levyAmount = weightTotal * levyShare;
        long weightBurden = weightTotal + levyAmount;
        return weightBurden;
    }
}
